src/apps/bridge/sensorController.cpp

```diff
@@ -4,12 +4,14 @@
 #include "bridgePowerController.h"
 #include "device_info.h"
 #include "reportBuilder.h"
+#include "softSensor.h"
 #include "sys_info_service.h"
 #include "sys_info_svc_reply_msg.h"
 #include "task_priorities.h"
 #include "util.h"
 
 #define DEFAULT_CURRENT_READING_PERIOD_MS 60 * 1000 // default is 1 minute: 60,000 ms
+#define DEFAULT_SOFT_READING_PERIOD_MS 500 // default is 500 ms (2 HZ)
```

Comment:
TODO: Once we have `bcmp_config` request reply, we should read this value from the modules.